### DISCUSSION TITLE
track Power Flow Calculation durations via GitHub Actions

### DISCUSSION THREAD

**Discussion by SteffenMeinecke** [e2nIEE/pandapower#2481] [Ideas]
I wonder whether it is possible and rational to track the power flow calculation durations via GitHub actions to detect when changes decrease the power flow calculation time.
Do GitHub actions have templates for that? Are measured timings in a github action run (at least a bit) comparable?
@vogt31337 @KS-HTK @rbolgaryn

**Comment by vogt31337** [comment]
If we can do it automatically, yes. I wanted to implement some Testing for the opf stuff.
